src/main/kotlin/com/sexysisters/tojserverv2/domain/teacher/domain/Image.kt

```diff
@@ -7,20 +7,13 @@ import javax.persistence.Embeddable
 import javax.validation.constraints.NotNull
 
 @Embeddable
-class Image private constructor (
+class Image (
```

Comment:
```suggestion
class Image(
```

코틀린 코드 컨벤션에 맞게 고쳐봤습니다.
생성자의 `()`는 클레스 이름과 붙여야 합니다.
그리고 Gradle `ktlinCheck` 한 번씩 돌려주세요!